packages/manager/modules/veeam-enterprise/src/dashboard/license/license.template.html

```diff
@@ -34,6 +34,7 @@
                 name="port"
                 data-ng-model="$ctrl.form.port"
                 required
```

Comment:
**question:** If the `input` is `readonly`, is it useful to keep the `required` attribute then?